.gitignore

```diff
@@ -39,3 +39,5 @@ build/
 # Virtual environments
 venv/
 env/
+
+poetry.lock
```

Comment:
Let's keep the lockfile in the repo boyeee

Reply:
OK 😞